privacy_policy/en/privacy-notice-product-android.md

```diff
@@ -78,6 +78,21 @@ If the “Send system report” option is enabled, the program will also send ad
 * Log file contents;
 * List of installed applications.
 
+### When AdGuard crashes
+
+Upon the first launch, users are asked whether they allow AdGuard to send automatic crash reports. If they do (and only if they do), the following information will be sent to our servers when the app crashes:
+
+* Device name and model;
+* OS version and kernel build;
+* ROOT status;
+* Unidentifyable Android ID;
+* Amounts of total and free RAM;
+* Screen orientation;
+* Screen resolution and density;
+* Application name and version.
```

Comment:
Also, the information related to the crash itself is being sent: error message and stack trace.

Reply:
You need to update the privacy policy date as well.